karma.conf.js

```diff
@@ -35,6 +35,12 @@ module.exports = function(config) {
             },
 
             {
+                pattern: 'test/lib/pptm.js',
```

Comment:
Could you just include with a regular `export * from './pptm.js'` to match the other test files?